### PR TITLE
Add an elixir language sample using the phoenix framework to the knat…

### PR DESCRIPTION
**Pull request by mikehelmick**
Add an elixir language sample using the phoenix framework to the knative serving samples.

### CODE REVIEW

File: serving/samples/helloworld-elixir/README.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
+# Hello World - Elixir Sample
+
+A simple web application written in Elixir using the phoenix framework.
```
Comment (evankanderson) [inline serving/samples/helloworld-elixir/README.md]: Link to https://elixir-lang.org/?

**Comment by dprotaso** [comment]
Trendy

File: serving/samples/helloworld-elixir/README.md (hunk 1 above)
Comment (mikehelmick) [reply serving/samples/helloworld-elixir/README.md]: done.

**Comment by mikehelmick** [comment]
I updated this to remove the npn lock file - this reduces the size considerably.

**Pull request merged**